src/umdsst/GetValues/GeoVaLsWrapper.f90

```diff
@@ -12,51 +12,61 @@ module geovals_wrapper
 
 contains
 
-subroutine geovals_wrapper_fill( c_key_geovals, c_locs, c_t1, c_t2, c_field) &
+subroutine geovals_wrapper_fill( c_key_geovals, c_locs, c_t1, c_t2, c_flds, nvar) &
        bind(c, name="geovals_wrapper_fill_f90")
     integer(c_int),  intent(inout) :: c_key_geovals
     type(c_ptr), value, intent(in) :: c_locs
     type(c_ptr),        intent(in) :: c_t1
     type(c_ptr),        intent(in) :: c_t2
-    type(c_ptr), value, intent(in) :: c_field
+    type(integer),      intent(in) :: nvar
```

Comment:
you should use `c_int` instead of `integer`. In all likelihood it won't make a difference because the two should be the same, but they don't have to be depending on the compiler

Reply:
Changed to integer(c_int) like c_key_geovals, removed write(*, *).